morecantile/models.py

```diff
@@ -705,6 +704,9 @@ def custom(
         TileMatrixSet
 
         """
+        # TODO is this correct to always assume?
+        geographic_crs = geographic_crs or crs.geodetic_crs
```

Comment:
```suggestion
        geographic_crs = geographic_crs or crs.geodetic_crs
        if not geographic_crs:
            raise ValueError("CRS doesn't have internal geodeticCRS/geographicCRS defined, please provide one")
```

Reply:
I think most CRS should have Geographic CRS but it's safe to raise an exception if not

Reply:
@AndrewAnnex what do you think?

Reply:
yep looks great to me, I'll commit the suggestion and cleanup the todo if it is still there